nerfstudio/viewer_beta/viewer.py

```diff
@@ -184,24 +185,30 @@ def nested_folder_install(folder_labels: List[str], element: ViewerElement):
             ]
         for c in self.viewer_controls:
             c._setup(self)
-        self.render_statemachine.start()
+
+    def get_camera_state(self, client):
```

Comment:
Can we make this: `def get_camera_state(self, client: viser.ClientHandle) -> CameraState:`